xml/snmp_sysdescr.xml

```diff
@@ -6446,8 +6446,8 @@ Copyright (c) 1995-2005 by Cisco Systems
     <example os.product="GE Series " os.version="V5 R3.10404.0">HiPath Wireless Access Controller - GE Series , System Version V5 R3.10404.0</example>
     <example os.product="V2110" os.version="08.01.01.0251">HiPath Wireless Access Controller - V2110, System Version 08.01.01.0251</example>
     <param pos="0" name="os.vendor" value="Siemens"/>
-    <param pos="0" name="os.device" value="HiPath"/>
-    <param pos="0" name="os.family" value="WAP"/>
+    <param pos="0" name="os.family" value="HiPath"/>
+    <param pos="0" name="os.device" value="WAP"/>
```

Comment:
This was a typo that switched the values between the `family` and `device` params.